build.user.properties.template

```diff
@@ -0,0 +1,21 @@
+nativity.dir=C:/Users/sync/Dev/streem/liferay-nativity
```

Comment:
is the only way to have this entire path here? we can't do a thing relative to user, or have this configurable?